L1Trigger/TrackFindingTracklet/src/TrackletProcessorDisplaced.cc

```diff
@@ -62,6 +65,8 @@ TrackletProcessorDisplaced::TrackletProcessorDisplaced(string name, Settings con
   constexpr int TCIndexMax = 191;
```

Comment:
Hard-wiring TCIndexMax is not pretty. Can this be derived from the wiring map or on the number of FPGA clock cycles per time-multiplexed period?